src/store/Company.js

```diff
@@ -26,7 +26,11 @@ const actions = {
     }
 };
 
-const getters = {};
+const getters = {
+    currentCompanyID(state) {
```

Comment:
Use camelCase variables.
```suggestion
    currentCompanyId(state) {
```